dockers/docker-fpm-frr/base_image_files/TS

```diff
@@ -28,10 +32,16 @@ if [[ ($NUM_ASIC -gt 1) ]]; then
         if [ $sub_role == 'FrontEnd' ]
         then
             echo -e "BGP"$asic" : \c"
-            if [[ -n "$TSA_STATE_UPDATE" ]]; then 
-                sonic-cfggen -a "$TSA_STATE_UPDATE" -w -n $NAMESPACE_PREFIX$asic
-                logger -t $1 -p user.info "BGP$asic: $log_msg"
-                echo "$log_msg"
+            if [[ -n "$TSA_STATE_UPDATE" ]]; then
+                current_tsa_state="$(sonic-cfggen -d -v BGP_DEVICE_GLOBAL.STATE.tsa_enabled -n $NAMESPACE_PREFIX$asic)"
+                if [[ $current_tsa_state  == $desired_tsa_state ]]; then
+                    echo "$err_msg"
+                    logger -t TSA -p user.info "$err_msg"
```

Comment:
`TSA` should be $1?

Reply:
thanks for catching this! fixed the issue